composer.json

```diff
@@ -396,6 +396,9 @@
             },
             "drupal/search_api_solr": {
                 "https://www.drupal.org/project/search_api_solr/issues/3449292#comment-15644442": "https://www.drupal.org/files/issues/2024-06-17/search-api-solr-fatal-error-declaration-of-sleep-3449292-12.patch"
+            },
+            "drupal/y_lb_article": {
+                "Add auto-sort Featured Articles by Publications Date": "https://git.drupalcode.org/project/y_lb_article/-/merge_requests/54.diff"
```

Comment:
merged, for deploy please use release, not patch - https://www.drupal.org/project/y_lb_article/releases/1.2.22  @shuklina